<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Reciever data</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Message ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="messageIdLineEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Key Index</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="keyIndexLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Receiver ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="receiverIdLineEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>T2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="t2LineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Pult ID</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="pultIdLineEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Insert Index</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="insertIndexLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Pult Receiver ID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="pultReceiverIdLineEdit"/>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Extract Index</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="extractIndexLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Battery</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" rowspan="2">
        <widget class="QLineEdit" name="batteryLineEdit"/>
       </item>
       <item row="4" column="2" rowspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Universal</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" rowspan="2">
        <widget class="QLineEdit" name="universalLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Key Code</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="keyCodeLineEdit"/>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>RSSI</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLineEdit" name="rssiLineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Packet #</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="packetNumberLineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QListWidget" name="receiverDataListWidget"/>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Key Data</string>
      </property>
      <widget class="QLineEdit" name="keyDataLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
